apps/wing/src/cli.ts

```diff
@@ -83,6 +126,7 @@ async function main() {
     )
     .option("-p, --plugins [plugin...]", "Compiler plugins")
     .hook("preAction", progressHook)
+    .hook("preAction", collectAnalyticsHook)
```

Comment:
let me try that again, I could not get "preSubcommand" to work last week (iirc I was not getting some command information I expected) but perhaps I was goofing something up. I would much rather have it once and done.

Reply:
Ah I see I was trying to parse the root command rather than the sub command when using that hook, which is why I thought data was missing. This works like a charm good call

Reply:
@MarkMcCulloh I reverted back to adding this as a "preAction" hook to each command. It appears that the lifecycle hook for "preSubcommand" does not get the options on the subcommand. The only options I can parse are the defaults.